public/app.js

```diff
@@ -33,13 +33,13 @@ var app = app || {};
     });
     if (localStorage.projectData && localStorage.eTag === eTag) {
       let data = JSON.parse(localStorage.projectData)
-      Project.buildProjects(data);
+      app.Project.buildProjects(data);
```

Comment:
Here and elsewhere in this module, you shouldn't need to use `app.` because `page` and `Project` are defined in this scope.